Makefile

```
@@ -81,6 +89,34 @@ yarn-setup:
+	yarn run e2e:install
+
+e2e-test:
+	@if [ "$(STOP_ON_FAILURE)" = "true" ]; then \
+		yarn run test:e2e:stop-on-failure -- $(E2E_ARGS); \
+	else \
+		yarn run test:e2e -- $(E2E_ARGS); \
+	fi
+
+# Usage: make e2e-test-one E2E_SPEC=tests/e2e/auth/auth.spec.js
+# Usage: make e2e-test-one E2E_SPEC=tests/e2e/auth/auth.spec.js STOP_ON_FAILURE=true
+e2e-test-one:
```

checkmake (0.3.2)

[warning] Target body for "e2e-test-one" exceeds allowed length of 5 lines (6).

(maxbodylength)